### PR TITLE
enh(release): Add Centreon Web 21.04.13 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target version

- [ ] 20.10.x (staging)
- [x] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] 22.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md [hunk 1]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.04.13
+
+#### Bug Fixes
+
+- [Chart] Fixed a PHP warning and removed a debug text appearing at the bottom of the page when modifying a curve template in "Monitoring  >  Performances  >  Curves"
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md:26]:
```suggestion
- [Chart] Fixed a PHP warning and removed a debug text appearing at the bottom of the page when modifying a curve template in "Monitoring > Performances > Curves"
```

File: i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md [hunk 2]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+- [UX] When the user does not have the "Action access" rights for the top counter, when he connects then he gets the top counter "skeleton style"
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md:27]:
```suggestion
- [UX] When users did not have the "Action access" rights for the top counter, the top counter was displayed "skeleton style". This has been fixed.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md [hunk 3]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+
+#### Security Fixes
+
+- [Administration] SQL Injections on ACL group listing
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md:31]:
```suggestion
- [Administration] SQL injections on ACL group listing
```

File: i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md [hunk 4]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Administration] SQL Injection on Knowledge Base configuration form
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md:32]:
```suggestion
- [Administration] SQL injection on Knowledge Base configuration form
```

File: i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md [hunk 5]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Administration] SQL Injections on LDAP listing
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md:33]:
```suggestion
- [Administration] SQL injections on LDAP listing
```

File: i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md [hunk 6]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] Command path traversal resulting in RCE on command edition form
+- [Configuration] SQL Injection on export configuration
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md:35]:
```suggestion
- [Configuration] SQL injection on export configuration
```

File: i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md [hunk 7]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] SQL Injections on SNMP traps edition form
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md:36]:
```suggestion
- [Configuration] SQL injections on SNMP traps edition form
```

File: i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md [hunk 8]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] SQL injection on Resources form
+- [Core] RCE in legacy PHP's class autoload
+- [Monitoring] SQL Injection on performance curve edition form
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-core.md:39]:
```suggestion
- [Monitoring] SQL injection on performance curve edition form
```

File: versioned_docs/version-21.04/releases/centreon-core.md [hunk 9]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.04.13
+
+#### Bug Fixes
+
+- [Chart] Fixed a PHP warning and removed a debug text appearing at the bottom of the page when modifying a curve template in "Monitoring  >  Performances  >  Curves"
```
Comment (cg-tw) [inline versioned_docs/version-21.04/releases/centreon-core.md:26]:
```suggestion
- [Chart] Fixed a PHP warning and removed a debug text appearing at the bottom of the page when modifying a curve template in "Monitoring > Performances > Curves"
```

File: versioned_docs/version-21.04/releases/centreon-core.md [hunk 10]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+- [UX] When the user does not have the "Action access" rights for the top counter, when he connects then he gets the top counter "skeleton style"
```
Comment (cg-tw) [inline versioned_docs/version-21.04/releases/centreon-core.md:27]:
```suggestion
- [UX] When users did not have the "Action access" rights for the top counter, the top counter was displayed "skeleton style". This has been fixed.
```

File: versioned_docs/version-21.04/releases/centreon-core.md [hunk 11]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+
+#### Security Fixes
+
+- [Administration] SQL Injections on ACL group listing
```
Comment (cg-tw) [inline versioned_docs/version-21.04/releases/centreon-core.md:31]:
```suggestion
- [Administration] SQL injections on ACL group listing
```

File: versioned_docs/version-21.04/releases/centreon-core.md [hunk 12]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Administration] SQL Injection on Knowledge Base configuration form
```
Comment (cg-tw) [inline versioned_docs/version-21.04/releases/centreon-core.md:32]:
```suggestion
- [Administration] SQL injection on Knowledge Base configuration form
```

File: versioned_docs/version-21.04/releases/centreon-core.md [hunk 13]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Administration] SQL Injections on LDAP listing
```
Comment (cg-tw) [inline versioned_docs/version-21.04/releases/centreon-core.md:33]:
```suggestion
- [Administration] SQL injections on LDAP listing
```

File: versioned_docs/version-21.04/releases/centreon-core.md [hunk 14]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] Command path traversal resulting in RCE on command edition form
+- [Configuration] SQL Injection on export configuration
```
Comment (cg-tw) [inline versioned_docs/version-21.04/releases/centreon-core.md:35]:
```suggestion
- [Configuration] SQL injection on export configuration
```

File: versioned_docs/version-21.04/releases/centreon-core.md [hunk 15]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] SQL Injections on SNMP traps edition form
```
Comment (cg-tw) [inline versioned_docs/version-21.04/releases/centreon-core.md:36]:
```suggestion
- [Configuration] SQL injections on SNMP traps edition form
```

File: versioned_docs/version-21.04/releases/centreon-core.md [hunk 16]
```diff
@@ -19,6 +19,25 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Configuration] SQL injection on Resources form
+- [Core] RCE in legacy PHP's class autoload
+- [Monitoring] SQL Injection on performance curve edition form
```
Comment (cg-tw) [inline versioned_docs/version-21.04/releases/centreon-core.md:39]:
```suggestion
- [Monitoring] SQL injection on performance curve edition form
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Comment by tuntoja** [comment]
another PR (same content) was opened here to avoid waiting for the RFC on PR #1413 
https://github.com/centreon/centreon-documentation/pull/1429

**Pull request closed without merging**